test/helpers/elasticsearch.ts

```diff
@@ -93,18 +93,47 @@ export async function populateIndex(
     const version = getESVersion(client);
     const mapping = dataType.toESMapping({ typeName: type, overrides, version });
 
-    await client.indices.create(
-        fixMappingRequest(
-            client,
-            {
-                index,
-                waitForActiveShards: 'all',
-                body: mapping,
-            },
-            false
-        )
-    );
+    try {
+        await client.indices.create(
```

Comment:
@jsnoble, if you call create twice you will get an unhelpful error, I added this try/catch to get a more helpful error.